### PR TITLE
CMake temporary fix

### PR DESCRIPTION
**Pull request by diivm**
- Part of splitting #8 (1/N)

### CODE REVIEW

File: bindings/python/CMakeLists.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ project(bindings)
 find_package(PCL COMPONENTS common REQUIRED)
 find_package(pybind11)
```
Comment (kunaltyagi) [inline bindings/python/CMakeLists.txt:9]: This needs `pybind11` installed, right? How can I use this cmake file without installing pybind11 globally?

Comment (diivm) [reply bindings/python/CMakeLists.txt:9]: Replace with add_subdirectory? 
https://pybind11.readthedocs.io/en/stable/compiling.html#find-package-vs-add-subdirectory

Comment (kunaltyagi) [reply bindings/python/CMakeLists.txt:9]: If it's just a swap and replace, keep it. Add a note to augment this with a CMake config option sometime later

Comment (diivm) [reply bindings/python/CMakeLists.txt:9]: Got it.

**Review by kunaltyagi — APPROVED** [review]

**Pull request merged**